interviews/README.md

```diff
@@ -0,0 +1,32 @@
+# Takehome Grading
+
+Hi!
+This is our grading rubric for takehome projects.
+Thanks for taking a look!
+
+These are the scores by which our engineering team evaluate backend takehome assignments.
+We _don't_ establish a hard minimum of score for any opening -- these scores are a general rubric, not a hard rule.
+
+## Rubrics
+
+- [Intern Backend Takehome](./frontend-takehome.md)
+- [Intern Frontend Takehome](./backend-takehome.md)
+
+## Why?
+
+We believe in openness and transparency in our engineering practices.
+There must be _some_ criteria by which we evaluate takehome submissions, and if candidates are meant to be taking them, we'd like them to be clear to everyone.
+
+We also welcome any feedback you have on these!
+
+## Takehome Assignment Specifications
```

Comment:
I'm not sure whether this section belongs here. This section seems to be about what makes a good takehome assignment, whereas the rest of this document describes how we evaluate takehome submissions.

Reply:
🤔 interesting. Filed a followup issue to discuss: #18